### ISSUE TITLE
When using git for windows clone nvda.wiki repository: fatal: unable to checkout working tree

### ISSUE THREAD

**Issue by dingpengyu** [nvaccess/nvda#10663]
hello 
I get the following error when using git for windows clone nvda.wiki repository:
### Steps to reproduce:
First open[nvda.wiki](https://github.com/nvaccess/nvda.wiki.git)
Then use git clone: https://github.com/nvaccess/nvda.wiki.git
After clone finish I get the following error:
error: invalid path 'Notes:-Adding-support-for-aria-attributes-to-browsers.md'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
### Actual behavior:
After clone finish I get the  error:

### Expected behavior:
clone finish
thanks

**Comment by Andre9642** [comment]
Maybe due to long path, try to take the clone on a small path.

**Comment by dingpengyu** [comment]
> Maybe due to long path, try to take the clone on a small path.
hello
I changed the clone directory and the following message still appears:
dpy@DESKTOP-G7I56QF MINGW64 /e/Source
$ git clone https://github.com/nvaccess/nvda.wiki.git
Cloning into 'nvda.wiki'...
remote: Enumerating objects: 571, done.
error: invalid path 'Notes:-Adding-support-for-aria-attributes-to-browsers.md'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
dpy@DESKTOP-G7I56QF MINGW64 /e/Source
thanks

**Comment by Andre9642** [comment]
OK. With Git 2.24.1.2 I've the same error. No problem with Git 2.23.0. Maybe that issue is related to the issue: https://github.com/git-for-windows/git/issues/2435
What version of Git are you using?
I think you should close this issue and (if it's not a duplicate) report it rather to the above mentioned repository.

**Comment by dingpengyu** [comment]
> OK. With Git 2.24.1.2 I've the same error. No problem with Git 2.23.0. Maybe that issue is related to the issue: [git-for-windows/git#2435](https://github.com/git-for-windows/git/issues/2435)
> What version of Git are you using?
> I think you should close this issue and (if it's not a duplicate) report it rather to the above mentioned repository.

git version 2.24.1.windows.2

**Comment by SamB** [comment]
I'm pretty sure it's the colon ...
